src/components/forms/SmsTemplateForm/SmsTemplateForm.tsx

```diff
@@ -40,14 +44,12 @@ export const CreateSmsTemplate: FC<PageComponentProps> = ({ initialState }) => {
 
   return (
     <div className='nhsuk-grid-row'>
-      <NHSNotifyBackButton formId='create-sms-template-back' action={action}>
-        <input type='hidden' name='smsTemplateName' value={smsTemplateName} />
-        <input
-          type='hidden'
-          name='smsTemplateMessage'
-          value={smsTemplateMessage}
-        />
-      </NHSNotifyBackButton>
+      <div className='nhsuk-back-link nhsuk-u-margin-bottom-6 nhsuk-u-margin-left-3'>
```

Comment:
Is it worth making this into a component? It's used twice. 

I also think on the review pages we are using the nhsuk `BackLink` might be worth replacing that with the new component if you want to make it.

Reply:
will move to using BackLink everywhere in the next PR as discussed